_aicspylibczi/Image.h

```diff
@@ -70,6 +69,7 @@ namespace pylibczi {
 
       virtual void loadImage(const std::shared_ptr<libCZI::IBitmapData>& bitmap_ptr_, size_t channels_) = 0;
 
+      virtual ~Image() { }
```

Comment:
the virtual destructor should be at the top of the inheritance hierarchy to avoid memory leaks.  at least SubblockSortable, or whatever base class it has.

Reply:
Thanks for catching that.